jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/test/java/org/jbpm/workbench/cm/client/list/CaseInstanceListSearchViewTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jbpm.workbench.cm.client.list;
+
+import org.jboss.errai.common.client.dom.MouseEvent;
+import org.jbpm.workbench.cm.client.newcase.NewCaseInstancePresenter;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CaseInstanceListSearchViewTest extends AbstractCaseInstanceListTest{
+    @Mock
+    CaseInstanceListSearchViewImpl actions;
+
+    @Mock
+    MouseEvent event;
+
+    @Mock
+    NewCaseInstancePresenter newCaseInstancePresenter;
+
+    @Before
+    public void init() {
+        super.init();
+        doAnswer((i) ->{
+            newCaseInstancePresenter.show();
```

Comment:
Not sure why you need these, the presenter should be a plain mock that you simply what to check if a method was called. The logic inside the presenter should be tested in the presenter test class.

Reply:
if call presenter.createCaseInstance, it will call newCaseInstancePresenter.show().
So I need add it.